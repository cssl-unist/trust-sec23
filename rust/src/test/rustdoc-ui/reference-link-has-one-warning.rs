// ignore-test
// check-pass

/// docs [label][with#anchor#error]
//~^ WARNING has an issue with the link anchor
pub struct S;
