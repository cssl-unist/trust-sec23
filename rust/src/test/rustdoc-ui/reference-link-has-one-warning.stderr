warning: `[with#anchor#error]` has an issue with the link anchor.
  --> $DIR/reference-link-has-one-warning.rs:3:18
   |
LL | /// docs [label][with#anchor#error]
   |                  ^^^^^^^^^^^^^^^^^ only one `#` is allowed in a link
   |
   = note: `#[warn(broken_intra_doc_links)]` on by default

warning: 1 warning emitted

